mlrun/track/trackers/mlflow_tracker.py

```diff
@@ -440,19 +439,17 @@ def _log_model(
         # Get the model info from MLFlow:
         model_info = mlflow.models.get_model_info(model_uri=model_uri)
 
-        # Prepare the archive path:
-        model_uri = pathlib.Path(model_uri)
-        archive_path = pathlib.Path(tmp_path) / f"{model_uri.stem}.zip"
-        if not os.path.exists(model_uri):
-            local_path = mlflow.artifacts.download_artifacts(
-                artifact_uri=str(model_uri)
-            )
-            model_uri = pathlib.Path(local_path)
+        local_model_path = mlflow.artifacts.download_artifacts(
+            artifact_uri=str(model_uri)
+        )
+        model_path = pathlib.Path(local_model_path)
```

Comment:
Please keep the headline comments.

```suggestion
        # Download the model and set the path to local path:
        local_model_path = mlflow.artifacts.download_artifacts(
            artifact_uri=str(model_uri)
        )
        model_path = pathlib.Path(local_model_path)
```